### DISCUSSION TITLE
Inconsitencies in relational fields

### DISCUSSION THREAD

**Discussion by mmmaxxxe** [directus/directus#14110] [Archive]
Hi there,
i use directus cloud 9.12.2 . I have a "main" collection with related tables (m2o or mtm or o2m)

![image](https://user-images.githubusercontent.com/102175921/175775979-b99b7e1f-e556-47bf-8de2-f4e6b53554f9.png)

The main table is correctly rendered and relations are correctly rendered to the app. (the yellow circles are on relationships)

if i use the related tables i have many inconstencies;

![image](https://user-images.githubusercontent.com/102175921/175776050-d49e1d95-1db6-4213-8001-c3bb71f31de6.png)

(yellow circles should renders the relation to the main table...)

But if i edit the related collection for that record it works just fine:

![image](https://user-images.githubusercontent.com/102175921/175776117-462640d7-34cf-45b0-b139-594ec64cb73a.png)

(the yellow circle is what i expect to see in the prevous table layout)

There are many incoonsistencies, don't know if it depends on the latest upgrade. It's the very same time that happens to me.

Thanks

**Comment by azrikahar** [comment]
This should be what's reported in #13836 (primarily the showing up in table layout part). It should be resolved by #13866 which will be included in the next release.
